mbed_tools_ci/generate_docs.py

```diff
@@ -37,11 +55,13 @@ def main() -> None:
         "--output_directory",
         help="Output directory for docs html files.",
         default=configuration.get_value(
-            ConfigurationVariable.DOCUMENTATION_DEFAULT_OUTPUT_PATH),
+            ConfigurationVariable.DOCUMENTATION_DEFAULT_OUTPUT_PATH
+        ),
     )
     args = parser.parse_args()
-    sys.exit(generate_docs(output_directory=args.output_directory))
+    output_directory = Path(args.output_directory)
+    sys.exit(generate_docs(output_directory=output_directory))
 
 
-if __name__ == '__main__':
+if __name__ == "__main__":
```

Comment:
We install this as a `console_script` right? If so, we don't need to call `main` here, so we could remove this. `setuptools` will add the `__name__` guard and call `main` for us in the  auto-generated `console_script` entry point.

Reply:
I would have to have a think about this one and look into what `setuptools` is doing, but even still I would be tempted to deal with all the scripts in the same way and make the change separate to this PR.